### ISSUE TITLE
Accessibility - Reactions not accessible

### ISSUE THREAD

**Issue by slafleche** [vanilla/vanilla#5947]
Set aria-hidden="true" on reactions flyout. It's not critical functionality.

**Comment by slafleche** [comment]
In PR: https://github.com/vanilla/internal/pull/1163
